src/plugins/RelightHelpers.js

```diff
@@ -0,0 +1,201 @@
+/**
+ *
+ * @param id
```

Comment:
Feels like we should add some documentation here since we've added the boilerplate

Reply:
I agree.